recipesearch/src/index.js

```diff
@@ -5,11 +5,14 @@ import App from './App';
 import * as serviceWorker from './serviceWorker';
```

Comment:
Given the current app setup, what does this line do?

Reply:
It was setup when I ran create react app.Let me remove it